Sources/Latte+Flows.swift

```diff
@@ -50,37 +45,55 @@ public extension Latte {
                 ).get()
 
                 let webViewRequest = LatteWebViewRequest(request: request)
-                let latteVC = LatteViewController(context: context, request: webViewRequest, webviewIsInspectable: webviewIsInspectable)
-                latteVC.delegate = self
-                viewController = latteVC
+                let latteVC = LatteViewController(request: webViewRequest, webviewIsInspectable: webviewIsInspectable)
+                latteVC.webView.delegate = self
+                latteVC.webView.load()
 
-                let result: LatteWebViewResult = try await withCheckedThrowingContinuation { next in
-                    latteVC.handler = { next.resume(with: $0) }
-                    context.pushViewController(latteVC, animated: true)
+                let _: Void = try await withCheckedThrowingContinuation { next in
+                    latteVC.webView.onReady = { _ in
+                        next.resume()
+                    }
+                    latteVC.webView.completion = { (_, result) in
+                        next.resume(with: result.map { _ in () })
+                    }
                 }
 
-                let userInfo = try await result.handle { _, completion in
-                    authgear.experimental.finishAuthentication(finishURL: result.finishURL, request: request, handler: completion)
+                let handle = LatteHandle<UserInfo> { completion in
+                    Task { await run() }
+                    @Sendable @MainActor
+                    func run() async {
+                        do {
+                            let result: LatteWebViewResult = try await withCheckedThrowingContinuation { next in
+                                latteVC.webView.completion = { (_, result) in
```

Comment:
If API consumer do some long operations before calling `wait`, the `completion` event may be missed. I think we may want to do this instead:
- setup two `Task` (ready & completion) before loading the webview
    - in the tasks, assign the callbacks to webview and wait for the result
- return control to API consumer when either of  ready/completion tasks is completed
- wait for completion task to complete when LatteHandle.wait is called.

Reply:
Changed LatteHandle to contain a Task. Basically it just wraps a Task.